nx-dev/ui-podcast/src/lib/podcast-list.tsx

```diff
@@ -0,0 +1,49 @@
+import { BlogPostDataEntry } from '@nx/nx-dev/data-access-documents/node-only';
+import { BlogAuthors } from '@nx/nx-dev/ui-blog';
+import Link from 'next/link';
+
+export interface PodcastListProps {
+  podcasts: BlogPostDataEntry[];
+}
+
+export function PodcastList({ podcasts }: PodcastListProps): JSX.Element {
+  return podcasts.length < 1 ? (
+    <></>
```

Comment:
Can you test before calling the component? It will simplify template.

Reply:
Or do we want to say something when there is no podcast? @juristr

Reply:
I would put something there, yes. Even though that's not gonna be the case as we're going to push this only once the first podcast edition is created, which Zack is in the process of doing. He has [a PR open](https://github.com/nrwl/nx/pull/26984), I just told him to fix a couple of things first before publishing

Reply:
Added an empty state, aka Stay tuned. @juristr lmk if the wording is fine and we can leave it as is.